.travis/apisix_cli_test.sh.bak

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env bash
```

Comment:
This file is useless?

Reply:
It maybe imported by the rebase, I'll clean it.